packages/notebook/src/notebooktools.ts

```diff
@@ -78,11 +78,15 @@ export class NotebookTools extends Widget implements INotebookTools {
     this.translator = options.translator || nullTranslator;
     this._trans = this.translator.load('jupyterlab');
     this._commonTools = new RankedPanel<NotebookTools.Tool>();
+    this._commonTools.title.label = this._trans.__('Common Tools');
     this._advancedTools = new RankedPanel<NotebookTools.Tool>();
+    this._advancedTools.id = 'advancedToolsSection';
     this._advancedTools.title.label = this._trans.__('Advanced Tools');
 
+    this._extendedTools = [];
+
     const layout = (this.layout = new PanelLayout());
-    layout.addWidget(this._commonTools);
+    layout.addWidget(new Collapser({ widget: this._commonTools }));
```

Comment:
In a follow-up, it can be interesting to switch to a `SidePanel` to get a better overall look and feel for the side bars.